webui/react/src/ioTypes.ts

```diff
@@ -136,6 +136,7 @@ const validationHistoryIoType = io.type({
 const trialSummaryIoType = io.type({
   hparams: io.any,
   id: io.number,
+  num_steps: io.number,
```

Comment:
**suggestion:** you can leave this to a separate PR but there is a `num_batches: io.number` available now.

Reply:
I added it in.